wled00/wled.h

```diff
@@ -284,7 +299,11 @@ WLED_GLOBAL char ntpServerName[33] _INIT("0.wled.pool.ntp.org");   // NTP server
 
 // WiFi CONFIG (all these can be changed via web UI, no need to set them here)
 WLED_GLOBAL char clientSSID[33] _INIT(CLIENT_SSID);
+WLED_GLOBAL char wifiEncryptionType[2]   _INIT("0");
 WLED_GLOBAL char clientPass[65] _INIT(CLIENT_PASS);
+WLED_GLOBAL char enterpriseAnonymousIdentity[65] _INIT(ENTERPRISE_ANONYMOUS_IDENTITY);
+WLED_GLOBAL char enterpriseIdentity[65] _INIT(ENTERPRISE_IDENTITY);
+WLED_GLOBAL char enterprisePass[65] _INIT(ENTERPRISE_PASS);
```

Comment:
Why not reuse clientPass? Consuming 180 bytes of RAM is quite stressful on ESP8266.

Reply:
Good point, I am now reusing clientPass.